apps/ledger-live-mobile/src/components/ServicesWidget/Protect/ActiveProtectState.tsx

```diff
@@ -9,9 +9,11 @@ function ActiveProtectState({ params }: { params: Record<string, string> }) {
 
   const { viewDetailsURI } = params || {};
 
+  const source = "ledgerlive://myledger";
```

Comment:
There are these links repeated everywhere in this PR. Maybe but these in a common place and export them ?